### ISSUE TITLE
Can I change embeddings['image'].shape from 768 to 1024?

### ISSUE THREAD

**Issue by dongfeicui** [PKU-YuanGroup/LanguageBind#18]
I want to use pretrained weights to inference, but I need embeddings['image'].shape from 768 to 1024.
How to do that?

**Comment by LinB203** [comment]
You can finetune by adding a projection layer.
Btw, we are tuning a  version of huge , which is 1024-dim.
